src/extension.ts

```diff
@@ -21,6 +21,8 @@ async function initializeExtension(operationId: string, context: vscode.Extensio
   initRecommendations(context);
   initMisc(context);
 
+  vscode.window.registerWebviewPanelSerializer('java.overview', new OverviewViewSerializer());
```

Comment:
Since `registerWebviewPanelSerializer()` will return a `Disposable`, we need to push it into the `context.subscriptions`

Reply:
Good suggestion!